mobile/src/full/java/org/openhab/habdroid/core/CloudMessagingHelper.java

```diff
@@ -38,21 +40,27 @@ public static void onNotificationSelected(Context context, Intent intent) {
         }
     }
 
-    public static @StringRes int getPushNotificationStatusResId() {
+    public static String getPushNotificationStatusResId(Context context) {
```

Comment:
Should rename this to reflect the new usage (drop the ResId suffix from the method name)